Alpaca.Markets/Messages/JsonStreamQuote.cs

```diff
@@ -33,16 +33,18 @@ internal class JsonStreamQuote : IStreamQuote
         [JsonIgnore]
         public DateTime Time { get; set; }
 
+        /// <summary>
+        /// Called when the json message is deserialized.
+        /// We adjust the milliseconds by dividing it by 1,000,000 to reduce the time stamp from nanoseconds to milliseconds
+        /// </summary>
+        /// <param name="context">The context.</param>
         [OnDeserialized]
-        internal void OnDeserializedMethod(
-            StreamingContext context)
+        internal void OnDeserializedMethod(StreamingContext context)
         {
 #if NET45
-            Time = DateTimeHelper.FromUnixTimeMilliseconds(Timestamp);
+            Time = DateTimeHelper.FromUnixTimeMilliseconds((Int64)(Timestamp / 1000000));
```

Comment:
Please, remove magic constant 100000 - introduce constant NANOSECONDS_IN_MILLISECOND and use it instead of this constant.